insights/parsers/tests/test_cups_ppd.py

```diff
@@ -33,6 +35,7 @@ def test_cups_ppd():
     cups_ppd_result = CupsPpd(context_wrap(CUPS_PPD, path='/etc/cups/ppd/test_printer1.ppd'))
     assert cups_ppd_result["PCFileName"] == '"ippeve.ppd"'
     assert cups_ppd_result["cupsFilter2"] == ['"application/vnd.cups-pdf application/pdf 10 -"', '"application/vnd.cups-postscript application/postscript 10 -"']
+    assert "test"not in cups_ppd_result
```

Comment:
You might want to add a blank space before the `not` in this line.

Reply:
@xiangce - Updated, please have a check.